### PR TITLE
Create Connection Serializer

### PR DESCRIPTION
**Pull request by SuhailaAhmed**

### CODE REVIEW

File: accounts/serializers/connection.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from accounts.models import connection
```
Comment (Suleiman99Hesham) [inline accounts/serializers/connection.py]: - it's better to import the model from the its files directly not from the `models` module
- like: `from accounts.models.connection import Connection`

File: accounts/serializers/connection.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+class ConnectionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = connection
```
Comment (Suleiman99Hesham) [inline accounts/serializers/connection.py]: - `Connection` instead of `connection`

**Review by Suleiman99Hesham — CHANGES_REQUESTED** [review]

**Comment by Suleiman99Hesham** [comment]
it seems that you created this branch from `GP2023-28`, not from `main` so please after branch `GP2023-28` is merged to `main`, merge `main` again with this branch to pull the changes of the `Connection` model, there are the needed steps after merging `GP2023-28`:

1. move to `main` branch on your local repo to update it for the remote repo -> `git switch main`
2. update `main` branch -> `git pull origin main`
3. check that you are on this branch again -> `git switch GP2023-29`
4. merge `main` to `GP2023-29` -> `git merge main`
5. fix conflicts if it exists
6. commit the merge -> `git commit -m "merge main branch into GP2023-29"`
7. push the changes to `origin/GP2023-29` -> `git push origin GP2023-29`

**Comment by SuhailaAhmed** [comment]
I had just added a new commit to this branch with the changes.

**Pull request merged**